convert2rhel/actions/pre_ponr_changes/handle_packages.py

```diff
@@ -37,15 +37,25 @@ def run(self):
         logger.task("Convert: List third-party packages")
         third_party_pkgs = pkghandler.get_third_party_pkgs()
         if third_party_pkgs:
-            logger.warning(
+            pkg_list = pkghandler.format_pkg_info(sorted(third_party_pkgs, key=self.extract_packages))
+            message = (
                 "Only packages signed by %s are to be"
                 " replaced. Red Hat support won't be provided"
-                " for the following third party packages:\n" % system_info.name
+                " for the following third party packages:\n%s" % (system_info.name, pkg_list)
```

Comment:
I have separated the `pkg_list` in this message, since it will be used to log as a warning, and the formatted table of pkgs would appear in the same line and with the color yellow for the user. 

https://github.com/oamg/convert2rhel/pull/888/files#diff-7bab0f1619d739dd7e18e3643a6814d444189bb25fc420206ae8e3a0b3ddde85R51-R52

We can leave this as-is and follow-up the change in the split four fields PR.

Reply:
Sounds good

Reply:
Note, in addition to the color, the formatting for the message in `add_message()` will differ.  `pkg_list` in this PR will be in tabular form.  In #888, it's a comma separated list based on `third_party_pkgs`.  We probably want to change that so it returns the formatted `pkg_list` as well.